test/unit/rubygem_searchable_test.rb

```diff
@@ -41,8 +41,8 @@ class RubygemSearchableTest < ActiveSupport::TestCase
 
   context 'filter' do
     setup do
-      example_1 = create(:rubygem, name: "example_1")
-      example_2 = create(:rubygem, name: "example_2")
```

Comment:
Something fishy is going on here and I don't know how can proceed further. After update, this test was failing because it didn't return any result for query: `example`. I checked the rubygem analyzer:
```
curl -XGET 'localhost:9200/rubygems-development/_analyze?analyzer=rubygem&pretty=true' -d 'example_1'
{
  "tokens" : [
    {
      "token" : "example",
      "start_offset" : 0,
      "end_offset" : 7,
      "type" : "word",
      "position" : 0
    },
    {
      "token" : "1",
      "start_offset" : 8,
      "end_offset" : 9,
      "type" : "word",
      "position" : 1
    }
  ]
}
```
As you can see, it is able to tokenize correctly (I have also added test for rubygems analyzer), however it was not able to find records.